req.go

```diff
@@ -56,6 +57,11 @@ func (req *reqSocket) Type() SocketType {
 	return req.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.